jOOR/src/main/java/org/joor/Reflect.java

```diff
@@ -211,6 +211,34 @@ public Reflect set(String name, Object value) throws ReflectException {
     }
 
     /**
+     * Set a final field value.
+     * <p>
+     * Version of {@link Field#set(Object, Object)} only for changing final variables. If the
+     * wrapped object is a {@link Class}, then this will set a value to a static
+     * member field. If the wrapped object is any other {@link Object}, then
+     * this will set a value to an instance member field.
+     * For restrictions of usage check: http://bit.ly/changing_final_fields_java

```

Comment:
For security reasons, we shouldn't have URL shortened URLs in the Javadoc. This should be replaced by the actual URL:
http://stackoverflow.com/questions/3301635/change-private-static-final-field-using-java-reflection